modules/admin_manual/pages/faq/index.adoc

```diff
@@ -1,30 +1,65 @@
 = Frequently Asked Questions
 :toc: right
 
+[[cannot-see-new-ldap-accounts]]
+== I cannot see new LDAP accounts
+
+Set up a new cronjob for your Apache user to run occ with the user:sync command regularly.
```

Comment:
@dercorn, would it be better to say "_web server user_" instead of "_Apache user_"? I appreciate that Apache is the main web server supported by ownCloud; but I'd suggest that it would be better to be vendor-agnostic, as users can use whatever they're prepared to support.

Reply:
@settermjd during recent discussions we decided that we want to narrow down the scope of mentioned alternatives. Yes, it is possible to run ownCloud with nginx and perhaps even lighttpd. But, we want to keep the majority of users on track with a single option, as we simply cannot effectively support all available options out there. Somebody has to actually answer all those support tickets.
Users of nginx and other web server software should know how to handle their stuff, can find answers  at central or should just go with Apache.